src/fidesctl/cli/commands/core.py

```diff
@@ -138,11 +140,20 @@ def parse(ctx: click.Context, manifests_dir: str, verbose: bool = False) -> None
 @click.command()
 @click.pass_context
 @manifests_dir_argument
+@click.option(
+    "--sync-new",
```

Comment:
Maybe `-o`, `--only` ?

Reply:
maybe `include-missing`?

Reply:
`fidesctl pull --include-missing`?

Reply:
I'm liking it, will update to match this suggestion ^